x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.test.tsx

```diff
@@ -796,6 +796,34 @@ describe('edit button', () => {
   });
 });
 
+describe('refresh button', () => {
+  it('should render alert details when clicked', () => {
+    const alert = mockAlert();
+    const alertType: AlertType = {
+      id: '.noop',
+      name: 'No Op',
+      actionGroups: [{ id: 'default', name: 'Default' }],
+      recoveryActionGroup,
+      actionVariables: { context: [], state: [], params: [] },
+      defaultActionGroupId: 'default',
+      minimumLicenseRequired: 'basic',
+      producer: ALERTS_FEATURE_ID,
+      authorizedConsumers,
+      enabledInLicense: true,
+    };
+
+    expect(
+      shallow(
+        <AlertDetails alert={alert} alertType={alertType} actionTypes={[]} {...mockAlertApis} />
+      )
+        .find(EuiButtonEmpty)
```

Comment:
Just a note that since we tagged the refresh button with `data-test-subj` inside the component, we can reference it using `[data-test-subj="refreshAlertsButton"]` instead of `EuiButtonEmpty` > `[name="refresh"]`